go-for-eat/screens/Profile/Profile.js

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import { Text, View, Image, TouchableOpacity, ScrollView, TextInput, KeyboardAvoidingView } from 'react-native';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+import { connect } from 'react-redux';
+import profileStar from '../../assets/icons/profile_star.png';
+import profileStarEmpty from '../../assets/icons/profile_star_empty.png';
+import profileEdit from '../../assets/icons/profile_edit.png';
+import profileSave from '../../assets/icons/profile_save.png';
+import s from './styles.js';
+import { updateUser } from '../../actions';
+
+class Profile extends Component {
+  state = {
```

Comment:
decide whether the state will always be inside the constructor or not. Avoid big differing linting styles.